src/events/EventsTypes.ts

```diff
@@ -1,17 +1,47 @@
 import { LawId } from "../laws"
 import { Citations } from "../citations"
 import { Context } from "../store"
-import { Details, Internals, Percent } from "../types"
+import { Details, Internals, Ratio } from "../types"
 import { Game } from "../game"
 
 export type EventDefinition = {
   title: string
   description: string
   apply(context: Context): void
-  probability(game: Game): Percent
+  /** A value used to determine if this event occurs now.
+   *
+   * Three possible values and their meanings are
+   * - Zero or less: The event may not occur now.
+   * - Larger 1: The event occurs now.
+   * - Between 0 and 1: Probability that the event occurs relative to others.
+   *
+   * Events will be processed as follows:
+   * - If some events return > 1, the one with the highest value will occur, otherwise
+   * - with a probability of {@link probabilityThatEventOccurs} no event occurs, otherwise
+   * - one of the events which returned a value between 0 and 1 occurs with its relative probability.
+   *
+   * The values returned by events (which returned between 0 and 1) may add to any value.
+   * An event returning a very large value will effectively reduce the probabilities of
+   * all other events. This means that an event returning 0.9 will most likely not occur
+   * with a probability of 90%, since other events (and {@link probabilityThatEventOccurs})
+   * will effectively reduce it.
+   * However, an event returning 0.9 will occur 9 times as often as an event returning 0.1.
+   *
+   * @param game The game data to use for determining the probability.
+   * @return The probability as described in the summary.
+   */
+  probability(game: Game): Ratio
   acknowledged?: boolean
   laws?: LawId[]
   citations?: Citations // TODO #79: Make mandatory
   details?: Details // TODO #79: Make mandatory
   internals?: Internals // TODO #79: Make mandatory
 }
+
+export const eventProbs = {
+  newYear: 2,
+  won: 3,
+  broke: 4,
+  lostElection: 5,
+  co2BudgetUsed: 6,
+}
```

Comment:
I understand that like that: The `eventProbs` are actually a priority. The `newYear` event is less significant as the fact that the `co2BudgetUsed`, so its value is less than the other one.
If that impression is correct, we should rename the variable into something like `eventPriorities`, to make the meaning more clear.

Reply:
Maybe we could even use the event IDs to make the reference more clear.

Reply:
The design is not optimal, there. These are special values for the event probabilities. Maybe `specialEventProbs`?

I am not at all convinced of the design to use a number return value of a function `probability()` and devide its range in several areas with completely different meaning, but i wanted to get it running fast to get back to the laws...

Reply:
Event IDs would be nice, but are unavailable at this point. We could still use them here "by hand" to make the correspondence more apparent.

Reply:
Ok, lets keep it like that.